TodoList/Src/TodoList.Contracts/Api.Services/ITodoListUrlHelper.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace TodoList.Contracts.Api.Services
+{
+    public interface ITodoListUrlHelper
+    {
+        Uri Link(Guid id);
```

Comment:
Rename the function to something more descriptive. For example:
```suggestion
        Uri GetItemLink(Guid id);
```

Reply:
c65e942